### DISCUSSION TITLE
Github claiming current chrome version doesnt support security keys

### DISCUSSION THREAD

**Discussion by ikiris** [community/community#23917] [New to GitHub]
<p>so… wtf?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/7/e76bf1b10fccec924c36aaf6db46ab3a2c1bdd0a.png" data-download-href="/uploads/short-url/x1fz1jBbQ7fix61HidLO5EHzITM.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4852950/181080004-df9b8517-a296-4d72-8599-831d335721d1.png" alt="image" data-base62-sha1="x1fz1jBbQ7fix61HidLO5EHzITM" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e76bf1b10fccec924c36aaf6db46ab3a2c1bdd0a_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e76bf1b10fccec924c36aaf6db46ab3a2c1bdd0a_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e76bf1b10fccec924c36aaf6db46ab3a2c1bdd0a_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e76bf1b10fccec924c36aaf6db46ab3a2c1bdd0a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 76.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tajmone** [comment]
<p>Did you check that there are no Chrome Extensions that might interfere with this (security, ad blockers, etc.). The fact that GitHub is reporting that Chrome doesn’t support the feature doesn’t necessarily mean it’s so, maybe it’s just not responding to some probe by GitHub, i.e. some other app is blocking the process (e.g. some anti-virus offer data protection features that prevent sensitive data from being passed to the browser unless you enable it for a specific domain, etc.).</p>

**Comment by ikiris** [comment]
<p>I have only very basic extensions installed.</p>
<p>betterttv<br>
ublock origin<br>
reddit enhancement suite<br>
enhanced steam<br>
steam database<br>
imagus</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="ikiris" data-post="3" data-topic="195127">
<div class="title">
<div class="quote-controls"></div>
 ikiris:</div>
<blockquote>
<p>I have only very basic extensions installed.</p>
</blockquote>
</aside>
<p>uBlock might be the culprit, it’s quite a powerful blocker:</p>
<ul>
<li><a href="https://github.com/gorhill/uBlock" class="inline-onebox" rel="noopener nofollow ugc">GitHub - gorhill/uBlock: uBlock Origin - An efficient blocker for Chromium and Firefox. Fast and lean.&lt;/ti</a></li>
</ul>
<p>from its repository:</p>
<blockquote>
<p>The default behavior of uBlock Origin when newly installed is to block ads, trackers and malware sites – through EasyList, EasyPrivacy, Peter Lowe’s ad/tracking/malware servers, Online Malicious URL Blocklist, and uBlock Origin’s own filter lists. […]</p>
<p>Furthermore, advanced mode allows uBlock Origin to work in default-deny mode, which mode will cause all 3rd-party network requests to be blocked by default, unless allowed by the user.</p>
</blockquote>
<p>You might want to try to disable this extension to test if that’s the problem.</p>

**Comment by ikiris** [comment]
<p>Yes, that would have been the first thing I tried.</p>
<p>I’m a SRE. I wouldn’t have posted if this was a problem on my end.</p>

**Comment by ikiris** [comment]
<p>Yep. Same here. My security keys work fine in other systems and my own keycloak infra, just github has decided to be broken for some reason.</p>

**Comment by ikiris** [comment, marked as answer]
<p>This github bug has been silently fixed apparently.</p>
